tools/travis-scripts/before-install.sh

```diff
@@ -103,6 +103,7 @@ fi
 if [ "$BUILD_TARGET" == "linux_cocos_new_test" ]; then
     download_deps
     install_linux_environment
+    python $COCOS2DX_ROOT/tools/appveyor-scripts/setup_android.py --ndk_only
```

Comment:
Why travis invoke appveyor related script?

Reply:
that's a pure python script to setup android environment